### PR TITLE
fix: GetSet implementation and Px options for Set cache operation

### PR DESCRIPTION
**Pull request by JigarJoshi**
## Describe your changes

- Added support for `PX` options for Set operation. 
- Added GetSet operation.

## How best to test these changes

- Integration test added.

### CODE REVIEW

File: server/services/v1/cache/runner.go [hunk 1]
```diff
@@ -221,14 +220,30 @@ func (runner *SetRunner) Run(ctx context.Context, tenant *metadata.Tenant) (*Cac
...
 		NX: runner.req.GetNx(),
 		XX: runner.req.GetXx(),
-		EX: time.Duration(runner.req.GetEx()) * time.Second,
+		EX: runner.req.GetEx(),
+		PX: runner.req.GetPx(),
 	}
+
+	var oldVal *internal.CacheData
+	if runner.req.GetGet() {
+		oldVal, _ = runner.cacheStore.Get(ctx, tableName, runner.req.GetKey(), &cache.GetOptions{
```
Comment (himank) [inline server/services/v1/cache/runner.go:229]: why not use `GetSet` API in store for this?

Comment (JigarJoshi) [reply server/services/v1/cache/runner.go:229]: GetSet doesn't accept expiration options.

Comment (JigarJoshi) [reply server/services/v1/cache/runner.go:229]: `setArgs` returns the old value but always in form of `string`.

Comment (himank) [reply server/services/v1/cache/runner.go:229]: Then can we have GetSet as a separate API that doesn't accept expiration?

Comment (JigarJoshi) [reply server/services/v1/cache/runner.go:229]: sure https://github.com/tigrisdata/tigris-api/pull/188

Comment (efirs) [reply server/services/v1/cache/runner.go:229]:
```
As of Redis version 6.2.0, this command is regarded as deprecated.
```

Comment (JigarJoshi) [reply server/services/v1/cache/runner.go]: updated the PR.

**Review by himank — APPROVED** [review]

**Pull request merged**